### DISCUSSION TITLE
Handling Buffer type

### DISCUSSION THREAD

**Discussion by NurbekGithub** [prisma/prisma#2118] [Q&A] [answered]
Can I use Buffer type for storing files in db with prisma?
Will I be able to query it?

**Comment by pantharshit00** [comment, marked as answer]
We don't support any `Buffer` or `Blob` database types at the moment. Support for them will come in the future via https://github.com/prisma/prisma/issues/446

You might be able to use `String` type for this but it depends on your use case

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
